server/src/main/scala/org/labrad/browser/StaticResourceHandler.scala

```diff
@@ -73,7 +73,7 @@ class StaticResourceHandler(
 
     // create http response and set headers
     val response = new DefaultFullHttpResponse(HTTP_1_1, OK, Unpooled.wrappedBuffer(bytes))
-    HttpHeaderUtil.setContentLength(response, bytes.length)
+    HttpUtil.setContentLength(response, bytes.length)
```

Comment:
The version of netty included with scalabrad changed, and this class name changed with it.